src/Claims/Factory.php

```diff
@@ -195,7 +193,7 @@ public function setRequest(Request $request)
     /**
      * Set the token ttl (in minutes).
      *
-     * @param  int  $ttl
+     * @param int $ttl
```

Comment:
In my opinion it would be better to just use the parameter and return types in the method definition. 

Currently we have a mix between type given in PHP doc and given as parameter. Due to the minimum required version of php 7.4 it should be fine to just have the types in the definition or am I wrong?

Is php-cs-fixer capable in transforming the code like this?

Reply:
> In my opinion it would be better to just use the parameter and return types in the method definition.

…

> Is php-cs-fixer capable in transforming the code like this?

No, that would be rector and I agree -> but different tool.

php-cs-fixer, they way it is also configured with the Symfony rule set, can and does remove _redundant_ phpdoc.

Achtung: if we to this, it's possible we start to create more incompatibilties with users of the previous jwt-auth who customize it. I already pointed out one such https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/56#issuecomment-974693969 so it we start to clean this up, we should imho target a new major version to not surprise people.

Make the 1.* series be the "compatible with jwt-auth with 0 changes" and refactor whatever we want/need in 2.*.